internal/core/ui/src/main/res/values/strings.xml

```diff
@@ -47,7 +47,7 @@
     <string name="recco_questionnaire_numeric_label_lb">lb.</string>
     <string name="recco_questionnaire_numeric_label_min">min.</string>
     <string name="recco_questionnaire_numeric_label_st">st.</string>
-    <string name="recco_questionnaire_outro_body">Velit officia consequat duis enim velit mollit. Exercitation veniam consequat sunt nostrud amet.</string>
-    <string name="recco_questionnaire_outro_title">Great! We’re done with the first part!</string>
+    <string name="recco_questionnaire_outro_body">Keep going! The more questionnaires you complete, the more accurate your health score will be. \nThen, Recco can make personalized recommendations for you to make real health improvements.</string>
+    <string name="recco_questionnaire_outro_title">Great! You finished the <xliff:g id="section_name">%1$s</xliff:g> section.</string>
```

Comment:
these sentences are not used in the app, @chemarubio I see that in `QuestionnaireOnboardingOutroScreen` we are using `recco_onboarding_outro_title` and `recco_onboarding_outro_body` instead, should we change them?